libs/design-system/icon-button/src/lib/icon-button-variant/icon-button-variant.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, input } from '@angular/core';
+
+/** Input options for icon button color */
+export type IconButtonVariant = 'light' | 'dark' | 'default';
+
+/**
+ * Directive for icon button variants (color)
+ */
+@Directive({
+  selector: '[hraIconButtonVariant]',
+  standalone: true,
+  host: {
+    '[class]': '"icon-button-variant-" + variant()',
+  },
+})
+export class IconButtonVariantDirective {
+  /** Input for icon button color variant */
+  readonly variant = input<IconButtonVariant>('default', { alias: 'hraIconButtonVariant' });
```

Comment:
Let's change the default value to dark. It seems to be the most commonly used version across the apps. Let's also change the string `default` to `color` since it is no longer the default